engine/pkg/cmd/executor/executor.go

```diff
@@ -116,6 +116,8 @@ func (o *options) complete(cmd *cobra.Command) error {
 
 	cmd.Flags().Visit(func(flag *pflag.Flag) {
 		switch flag.Name {
+		case "name":
```

Comment:
What's the difference of 'Name' and 'uuid' for executor?

Reply:
It seems redundant. Maybe we can remove one of them.
This PR doesn't change any logic, will leave a TODO comment.